packages/manager/apps/hub-react/src/components/hub-order-tracking/HubOrderTracking.component.tsx

```diff
@@ -165,37 +174,59 @@ export default function HubOrderTracking() {
                 )}
               />
             </Suspense>
-
-            <div className="mb-6 flex justify-center gap-3 items-center flex-wrap">
-              <OsdsText
-                level={ODS_THEME_TYPOGRAPHY_LEVEL.body}
-                hue={ODS_TEXT_COLOR_HUE._800}
-                color={ODS_THEME_COLOR_INTENT.primary}
-                className="block font-bold mr-1"
-              >
-                {format(new Date(currentStatus.date))}
-              </OsdsText>
-              <OsdsText
-                level={ODS_THEME_TYPOGRAPHY_LEVEL.body}
-                hue={ODS_TEXT_COLOR_HUE._800}
-                color={ODS_THEME_COLOR_INTENT.primary}
-                className="block mr-1"
+          )}
+          <div className="mb-6 flex justify-center gap-3 items-center flex-wrap">
+            {isLoading || isLastOrderLoading ? (
+              <OsdsSkeleton
+                data-testid="order_info_skeleton"
+                className="font-bold text-right flex flex-col items-center justifier-center"
+                inline
+                size={ODS_SKELETON_SIZE.sm}
+              />
+            ) : (
+              <>
+                <OsdsText
+                  level={ODS_THEME_TYPOGRAPHY_LEVEL.body}
+                  hue={ODS_TEXT_COLOR_HUE._800}
+                  color={ODS_THEME_COLOR_INTENT.primary}
+                  className="block font-bold mr-1"
+                >
+                  {format(new Date(currentStatus.date))}
+                </OsdsText>
+                <OsdsText
+                  level={ODS_THEME_TYPOGRAPHY_LEVEL.body}
+                  hue={ODS_TEXT_COLOR_HUE._800}
+                  color={ODS_THEME_COLOR_INTENT.primary}
+                  className="block mr-1"
+                >
+                  {t(`order_tracking_history_${currentStatus.label}`)}
+                </OsdsText>
+                <OsdsIcon
+                  size={ODS_ICON_SIZE.xxs}
+                  color={ODS_THEME_COLOR_INTENT.text}
+                  name={
+                    !ERROR_STATUS.includes(currentStatus.label) &&
+                    !isWaitingPayment
+                      ? ODS_ICON_NAME.OK
+                      : ODS_ICON_NAME.CLOSE
+                  }
+                ></OsdsIcon>
```

Comment:
**question:** After loading the response, 2 text box and an icon is displayed, does the skeleton match the final layout?